### ISSUE TITLE
Schema not propagating correctly (Postgres)

### ISSUE THREAD

**Issue by scrimshander** [airbnb/caravel#325]
I have been able to successfully connect to a Postgres db, in a non-public schema, and configure a table. However, when I create a slice or browse the table I get the following error, which appears to be an issue with explicit SQL not bringing along the schema name. The FROM statement should reference `schemaname.tablename` and is only referencing `tablename`. I am using the metadata_params JSON to indicate the schema.

```ProgrammingError: (psycopg2.ProgrammingError) relation "tablename" does not exist
LINE 2: FROM tablename
             ^
 [SQL: "SELECT fieldname AS fieldname, COUNT(*) AS count \nFROM tablename \nWHERE date_as_of >= '2015-04-11 12:30:48.000000' AND date_as_of <= '2016-04-11 12:30:48.000000' GROUP BY fieldname ORDER BY COUNT(*) DESC \n LIMIT 50000"]```

**Comment by alanmcruickshank** [comment]
I can confirm I'm getting the same behaviour. Appears to be raising the error at:
`payload = obj.get_json()`
of method:
`def explore(self, datasource_type, datasource_id):`
in class:
`class Caravel(BaseView):`

I'm likewise using metadata_params JSON to indicate schema.

**Comment by mistercrunch** [comment]
Ok, seems like I'll have to install a postgres install to get to the bottom of this. In the meantime, I think the workaround is to connect with a user that matches the schema name. Please confirm that this works, though it should be a temporary workaround only as we'll address the issue.

**Comment by mistercrunch** [comment]
Anybody interested in testing this branch? This should address all the issues.
https://github.com/airbnb/caravel/pull/330

**Comment by scrimshander** [comment]
Will do so now.

**Comment by scrimshander** [comment]
I'm a newbie to caravel entirely, and may just be running into issues with trying to test a new branch, but I am seeing references to `static/assets/javascripts/dist` where no `/dist` exists, causing 404s when I am trying to test `/explore/table/...`

**Comment by xrmx** [comment]
Looks like this has been fixed in #330 and #360. Closing.
